### PR TITLE
Bumping kubernetes client to 11.0.0

### PR DESCRIPTION
**Pull request by yue9944882**

### CODE REVIEW

**Comment by ryanjbaxter** [comment]
thanks!

**Comment by ryanjbaxter** [comment]
@yue9944882 let me know when this is ready to review

**Comment by yue9944882** [comment]
@ryanjbaxter i think it's ready for review, but the CI job is flaking due to "Connection Reset", can you help re-run the test?

**Comment by wind57** [comment]
@yue9944882 this is not going to work either way. you updated to `2.0.1-SNAPSHOT` but for example the image is missing here : https://hub.docker.com/r/springcloud/spring-cloud-kubernetes-configuration-watcher/tags?page=1&ordering=last_updated . We need indeed Ryan's help. I would also appreciate an explanation how this is possible... and how that image is pushed.

**Comment by yue9944882** [comment]
> you updated to 2.0.1-SNAPSHOT but for example the image is missing here

i thought the image is pulled from dockerhub before, but actally it is supposed to be built locally from `spring-cloud-kubernetes-controllers/spring-cloud-kubernetes-configuration-watcher` by running mvn install. that's why i refactored the script for integration test

**Comment by wind57** [comment]
oh! I see you changed `run.sh` indeed... 

you can always push a dummy commit (like a new line somewhere) to re-trigger the build, btw.

**Comment by yue9944882** [comment]
@ryanjbaxter @wind57 hooray it passed!

**Comment by ryanjbaxter** [comment]
@yue9944882 can you fix the merge conflicts?

File: spring-cloud-kubernetes-integration-tests/run.sh [hunk 1]
```diff
@@ -100,23 +103,24 @@ main() {
 	# curl -L https://istio.io/downloadIstio | sh -
 	#"${ISTIOCTL}" install --set profile=demo
 
-	cd $CURRENT_DIR
+	# building required images for running integration tests
+	cd $CURRENT_DIR/../spring-cloud-kubernetes-controllers/spring-cloud-kubernetes-configuration-watcher
```
Comment (ryanjbaxter) [inline spring-cloud-kubernetes-integration-tests/run.sh:107]: The build should be run before running this script so this is unnecessary

File: spring-cloud-kubernetes-commons/pom.xml [hunk 2]
```diff
@@ -39,6 +39,10 @@
 			<groupId>org.springframework.cloud</groupId>
 			<artifactId>spring-cloud-loadbalancer</artifactId>
 		</dependency>
+		<dependency>
```
Comment (ryanjbaxter) [inline spring-cloud-kubernetes-commons/pom.xml:42]: Why is this added?

**Review by ryanjbaxter — CHANGES_REQUESTED** [review]

**Pull request merged**